resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -0,0 +1,225 @@
+    are then annotated. Finnally, annotated novel junctions are filtered in a separate output file.
+    More information can be found in the [regtools manual](https://regtools.readthedocs.io/en/latest/).
+  input:
+    - name: genome
+      label: Reference genome
+      type: data:genome:fasta
+    - name: annotation
+      label: Reference genome annotation (GTF)
+      type: data:annotation:gtf
+    - name: alignment_star
+      label: STAR alignment
+      type: data:alignment:bam:star
+      description: >
+        Splice junctions detected by STAR aligner (SJ.out.tab STAR output file). Please provide one
+        input 'Splice junctions extracted by STAR aligner' or 'Aligned reads' by any aligner or
+        directly 'Junctions in BED12 format'.
+      required: false
+    - name: alignment
+      label: Alignment
+      type: data:alignment:bam
+      required: false
+      description: >
+        Aligned reads from which splice junctions are going to be extracted. Please provide
+        one input 'Splice junctions detected by STAR aligner' or 'Aligned reads' by any aligner or
+        directly 'Junctions in BED12 format'.
+    - name: input_bed_junctions
+      label: Junctions in BED12 format
+      type: data:bed
+      required: false
+      description: >
+        Splice junctions in BED12 format. Please provide one input 'Splice junctions detected by
+        STAR aligner' or 'Aligned reads' by any aligner or directly 'Junctions in BED12 format'.
+  output:
+    - name: novel_splice_junctions
+      label: Table of annotated novel splice junctions
+      type: basic:file
+    - name: splice_junctions
+      label: Table of annotated splice junctions
+      type: basic:file
+    - name: bed
+      label: Splice junctions in BED format
+      type: basic:file
+    - name: bigbed_igv_ucsc
+      label: Splice junctions in BigBed format
+      type: basic:file
+      required: false
+    - name: tbi_jbrowse
+      label: Bed tbi index for JBrowse
+      type: basic:file
+    - name: species
+      label: Species
+      type: basic:string
+    - name: build
+      label: Build
+      type: basic:string
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+
+      {% if not alignment_star and not alignment and not input_bed_junctions %}
+        re-warning "Please define STAR alignment or BAM file or Junctions in BED12 format."
+      {% endif %}
+
+      {% if (alignment_star and alignment)
+        or ( alignment_star and input_bed_junctions)
+        or (alignment and input_bed_junctions ) %}
+        re-error "Please define only STAR alignment or BAM file or Junctions in BED12 format."
+      {% endif %}
+
+      {% if not genome.species == annotation.species %}
+        re-error \
+          {{ ('Species of genome (%s) and annotation (%s) do not match. Please provide genome ' ~
+             'and annotation with the same species.')|format(genome.species, annotation.species) }}
```

Comment:
Maybe it would be more correct: `with the same species` -> `of the same species`

Reply:
ok